docs/contivctl.md

```diff
@@ -0,0 +1,227 @@
+## contivctl
+
+contivctl command line tool lets you add/modify/delete objects into contiv object model using REST api.
+
+*Note:*
+`contivctl` Command can be found under `$GOPATH/bin` directory (`/opt/gopath/bin` in vagrant VMs) after a build. 
+Currently, `contivctl` needs to be run on the same host where `netmaster` is running
+
+## Usage
+
```

Comment:
@shaleman @jainvipin  

I think if we do consider putting an example deployment configuration in this document then may be we should also consider renaming this file to some thing like `ConfiguringPolicies.md` or something? otherwise it might be hard for the users to relate the contents from the file-name.

BTW while looking at `docs` directory, I think `docs/ConfigDetails.md` is now obsolete, is that correct?